website/docs/reference/opspec/index.md

```diff
@@ -2,96 +2,29 @@
-                    - [pullCreds](op-directory/op/call/container/image.md#pullcreds)
-                - [name](op-directory/op/call/container/index.md#name)
-                - [ports](op-directory/op/call/container/index.md#ports)
-                - [sockets](op-directory/op/call/container/index.md#sockets)
-                - [workDir](op-directory/op/call/container/index.md#workdir)
-            - [op](op-directory/op/call/op.md)
-                - [inputs](op-directory/op/call/op.md#inputs)
-                - [outputs](op-directory/op/call/op.md#outputs)
-                - [pullCreds](op-directory/op/call/op.md#pullcreds)
-                - [ref](op-directory/op/call/op.md#ref)
-            - [parallel](op-directory/op/call/index.md#parallel)
-            - [parallelLoop](op-directory/op/call/parallel-loop.md)
-                - [range](op-directory/op/call/parallel-loop.md#range)
-                - [run](op-directory/op/call/parallel-loop.md#run)
-                - [vars](op-directory/op/call/parallel-loop.md#vars)
-            - [serial](op-directory/op/call/index.md#serial)
-            - [serialLoop](op-directory/op/call/serial-loop.md)
-                - [range](op-directory/op/call/serial-loop.md#range)
-                - [run](op-directory/op/call/serial-loop.md#run)
-                - [until](op-directory/op/call/serial-loop.md#until)
-                - [vars](op-directory/op/call/serial-loop.md#vars)
-        - [version](op-directory/op/index.md#version)
-    - [icon.svg](op-directory/index.md#iconsvg)
-
-## Types
-Values in opspec are typed.
-
-Reference:
+
+Opspec (a portmanteau of operation specification) is a language designed to portably and fully define ops (operations).
+
+An op is any directory containing a [valid opspec file with the name `op.yml`](op.yml/index). By default, the opctl cli will look for ops in the `.opspec/` directory in your current working directory, but ops can be defined in any directory.
+
+An op directory may contain other files and directories. These files are considered "embedded" within the op and can be referenced during execution as executable code, as data, or for whatever purpose you have.
+
+If the root of an op directory contains a `icon.svg` file (must be [SVG 1.1](https://www.w3.org/TR/SVG11) with a 1:1 aspect ratio), it will be used when displaying the op within a UI.
+
+## Call graph
+
+As an op runs, opctl parses [`op.yml`](op.yml/index) files to create a dynamic call graph tracking the different running containers as the are started and exit. The call graph is defined declaritively using [call objects](op.yml/call/index), which include sequencing and conditional support.
+
+## Data flow
+
+Opctl has first class support for data flow into, between, and out of ops and their containers. Opspecs receive data from [`inputs`](op.yml#inputs) and return data through [`outputs`](op.yml#outputs). Within an opspec, data is referenced with a named variable that contains a typed value.
```

Comment:
Maybe consider `with a named variable that contains a typed value` => `via named variables containing typed values` (could also consider linking typed to the types section)

Reply:
Good rewording. Given this flows directly into the types section I won't add a link